budget-manager/src/app/app.component.html

```diff
@@ -5,25 +5,20 @@
         <mat-icon>menu</mat-icon>
     </mat-toolbar>
 
-    <mat-grid-list cols="2" rowHeight="42.5vh" gutterSize="40px" class="grid-container">
-        <mat-grid-tile>
-            <mat-card class="fill">
-
+    <mat-grid-list cols={{colsNo}} (window:resize)="onResize($event)" rowHeight="26vh" gutterSize="20px" class="grid-container">
```

Comment:
use brackets for dynamic inputs
```suggestion
    <mat-grid-list [cols]="colsNo" (window:resize)="onResize($event)" rowHeight="26vh" gutterSize="20px" class="grid-container">
```

Reply:
fixed